oaknational/Oak-Web-Application
# [CUR-276]: added sub-bullets for subject principles

## Description

- Added sub-bullets if the unicode character is present in a subject principle
- The subject principles would just all format in one bullet
- Updated subject principles cars colour to mint with updated design

## How to test

1. Go to https://deploy-preview-1963--oak-web-application.netlify.thenational.academy/teachers/curriculum/music-secondary-aqa/overview
2. Click on Music Secondary
3. You should see sub-bullets for one of the subject principles

## Screenshots

How it used to look (delete if n/a):
<img width="1371" alt="Screenshot 2023-10-03 at 13 01 03" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/2086ddda-038c-44c6-a1b4-5fd5e2ff07ce">

How it should now look:
<img width="1219" alt="Screenshot 2023-10-03 at 13 41 38" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/09a238b8-0ba9-43fc-8721-754073551cfc">


## Checklist
